pkg/apiserver/registry/inventory/rest.go

```diff
@@ -176,9 +180,14 @@ func (r *REST) ConvertToTable(ctx context.Context, obj runtime.Object, tableOpti
 	}
 	var err error
 	table.Rows, err = metatable.MetaToTableRow(obj,
-		func(obj runtime.Object, m metav1.Object, name, age string) ([]interface{}, error) {
+		func(obj runtime.Object, _ metav1.Object, _, _ string) ([]interface{}, error) {
 			vpc := obj.(*runtimev1alpha1.Vpc)
 			return []interface{}{vpc.Name, vpc.Info.CloudProvider, vpc.Info.Region}, nil
 		})
 	return table, err
 }
+
+func (r *REST) Watch(ctx context.Context, options *internalversion.ListOptions) (watch.Interface, error) {
+	key, label, field := networkpolicy.GetSelectors(options)
```

Comment:
Are you importing networkpolicy package, only to access this function?
if that is the case, do not import the package and define a helper function locally

Reply:
solved